### DISCUSSION TITLE
New races some might already be added

### DISCUSSION THREAD

**Discussion by Mythicbeanz** [jacobfortune/Songs-Of-War-Addon#7] [Ideas]
Jaggathen
Glacians
and like the reaper people but make the scythe have loyalty
Undead/necromancers

also plzzzzzz make the enderknight dragons bigger?

**Comment by lesleyespire** [comment]
Excuse me this project is not cancelled I’m still doing stuff just not on discord

**Comment by lesleyespire** [comment]
I will consider these suggestions, I actually finished jaggathan but it’s probably not posted here. I can make some Reaper armour if you like but I don’t think loyalty will work in the scythe. I might make a third stage to the ender dragon were it is in its adult form.
